service/gcs/bridge/bridge.go

```diff
@@ -24,7 +24,7 @@ import (
 // NotSupported represents the default handler logic for an unmatched
 // request type sent from the bridge.
 func NotSupported(w ResponseWriter, r *Request) {
-	w.Error("", gcserr.WrapHresult(errors.Errorf("bridge: function not supported, header type: 0x%x", r.Header.Type), gcserr.HrNotImpl))
+	w.Error("", gcserr.WrapHresult(errors.Errorf("bridge: function not supported, header type: 0x%x", r.Header.Type), gcserr.HrVmcomputeUnknownMessage))
```

Comment:
I think we can only return the 'proper' code in the v4 version case right? Otherwise this is a breaking change to the HCS?

Reply:
Also, did you run the unit tests for the bridge? I would guess I had an invalid message test that would fail now.

Reply:
Yeah, that's kinda what my bold text comment in the description means :). I mostly just wanted to get this PR up and then you or I could make that necessary change after protocol negotiation is added.

Reply:
And, yeah, I updated the unit tests so they should pass.